app/src/main/java/com/itsaky/androidide/activities/editor/BaseEditorActivity.kt

```diff
@@ -201,9 +205,18 @@ abstract class BaseEditorActivity :
       return
     }
 
-    Snackbar.make(binding.realContainer, string.msg_action_open_application, Snackbar.LENGTH_LONG)
-      .setAction(string.yes) { IntentUtils.launchApp(this, packageName) }
-      .show()
+    flashbarBuilder(duration = DURATION_INDEFINITE)
```

Comment:
@itsaky, do I leave the duration undefined and create a variable to check if the Flashbar is already visible, or do I leave the Flashbar with a short duration?

Reply:
>  do I leave the duration undefined

I think, yes. `DURATION_INDEFINITE` should be fine.

> and create a variable to check if the Flashbar is already visible

If you plan to dismiss the Flashbar if it is already visible, you'll probably keep a reference to the shown Flashbar. In that case, please make sure that you release the Flashbar instance properly.

Reply:
Okay